GatekeeperStudyHall/Assets/Scripts/RollButtonAction.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RollButtonAction : MonoBehaviour
+{
+    public void DiceRoll() {
+        // Random.Range returns a float which is then truncated to an int so the range will be 1 - 6
```

Comment:
I know this is from your main merge, but I'm noting here that the comment is wrong. Random.Range has an integer overload that does, in fact, return an integer. The code is correct, the comment is wrong. Max bound is exclusive, so `1, 6` is correct.

Reply:
Ok this should do it, thanks
```
public void DiceRoll() {
    // Random.Range(int, int) includes the lower value and excludes the higher value,
    // so the range will be 1 - 6
    int rollOutcome = Random.Range(1,7);
    Debug.Log(rollOutcome);
}
```